filters/flowid/generator.go

```diff
@@ -0,0 +1,11 @@
+package flowid
```

Comment:
Question, have you considered that this interface is something that the code in filter.go expects? Maybe, it could moved to there. I still think that it doesn't deserve its own file.